controllers/machinehealthcheck_controller_test.go

```diff
@@ -522,43 +518,22 @@ func TestClusterToMachineHealthCheck(t *testing.T) {
 	// BEGIN: Set up test environment
 	g := NewWithT(t)
 
-	testEnv = &envtest.Environment{
-		CRDs: []runtime.Object{
-			external.TestGenericBootstrapCRD,
-			external.TestGenericBootstrapTemplateCRD,
-			external.TestGenericInfrastructureCRD,
-			external.TestGenericInfrastructureTemplateCRD,
-		},
-		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases")},
-	}
```

Comment:
Is this change required for the purpose of this PR?

Reply:
Not strictly but I thought it was a good cleanup

Reply:
Sounds good, in general I'd suggest to split changes in multiple steps, it's easier to look at changes if they're logically separated in multiple PRs

Reply:
Yeah I'll pull it out into a separate PR